var/spack/repos/builtin/packages/blaze/package.py

```diff
@@ -16,8 +16,11 @@ class Blaze(Package):
     """
 
     homepage = "https://bitbucket.org/blaze-lib/blaze/overview"
-    url      = "https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.4.tar.gz"
+    url      = "https://bitbucket.org/blaze-lib/blaze/downloads/blaze-3.5.tar.gz"
+    git      = "https://bitbucket.org/blaze-lib/blaze.git"
 
+    version('master', branch='master', submodules=True, preferred=False)
```

Comment:
`preferred=False` is False by default. You don't need to add this, Spack will choose the latest numeric release by default.

Reply:
I actually went to look that up and got distracted by other parts of the documentation.  Thanks for the reminder :-D.

I'll update the PR tomorrow.

Reply:
Also there are no submodules so I will turn that off, too.